build.gradle

```diff
@@ -195,7 +200,9 @@ dependencies {
     // route all requests to java.util.logging to SLF4J (which in turn routes to tinylog)
     implementation 'org.slf4j:jul-to-slf4j:2.0.7'
 
-    implementation 'de.undercouch:citeproc-java:3.0.0-beta.2'
+    implementation('de.undercouch:citeproc-java:3.0.0-beta.2') {
+        exclude group: 'org.antlr'
```

Comment:
This needs to be carefully tested I remembere that different asm versions were producing troubles when generating CSL styles at runtime

Reply:
My assumptions:

1. The direct dependencies of `build.gradle` are loaded first
2. In case a library with the same group and name exists, it is ignored.